### DISCUSSION TITLE
There should be an option that mirror's the -F/--classify option for ls

### DISCUSSION THREAD

**Discussion by jmcantrell** [gokcehan/lf#1464] [Ideas]
This is a common (ls, exa, tree, ranger, probably others) option that I think is very helpful when scanning the listing.

Depending on the path, a character is appended (with the default foreground color) depending on what the path is:

- `@` means symbolic link (or that the file has extended attributes)
- `*` means executable
- `=` means socket
- `|` means named pipe
- `>` means door (not sure I've ever encountered this)
- `/` means directory

This is similar to icons, but a simpler alternative for those that just want a simple classification that's good enough without thinking about font support or every type of file that could be encountered.

Please consider this 🙏

**Comment by gokcehan** [comment]
@jmcantrell Thank you for your interest in the project and the proposal. I have thought about this feature a few times before in the past. While I think it is a reasonable feature, I should say we already have quite a lot of complexity in [the code for the directory drawing](https://github.com/gokcehan/lf/blob/9ab2c2ec6f41485f30a6f41acc507a8336e6e9af/ui.go#L356), so I'm not sure how others might think. Still I think the idea is worth considering, so I will move the discussion to the issues.

**Comment by CatsDeservePets** [comment]
@jmcantrell 
If you are willing to compile `lf` yourself, you can take the code out of my pull request that wasn't merged:
- https://github.com/gokcehan/lf/pull/1992
